docker/build/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+
+  python-dev:
+    build:
+      context: ../../
+      dockerfile: docker/images/python/dev/Dockerfile
+    image: fidals/se:dev
+
+  python-prod:
+    build:
+      context: ../../
+      dockerfile: docker/images/python/prod/Dockerfile
```

Comment:
maybe we can split dockerfiles themself instead of subdirectoring:
`docker/images/python/Dockerfile.(prod|dev)`

Reply:
We can, but what benefits it will give us?

Reply:
@artemiy312 , we'll avoid unnecessary subdirectoring. It seems enough benefit for me =)